ci-scripts/copy_to_archive

```diff
@@ -15,60 +14,69 @@ ARCHIVE_DIR="$2"
 IMAGE_DIR="${BUILD_DIR}/tmp/deploy/images"
 SDK_DIR="${BUILD_DIR}/tmp/deploy/sdk"
 LOG_DIR="${BUILD_DIR}/tmp/log"
-
-INTEL_ARCH="intel-corei7-64"
-ARP_ARCH="arp"
-RPI_ARCH="raspberrypi3"
-QEMU_ARCH="qemux86-64"
-JETSON_ARCH="jetson-tx2"
+LICENSE_DIR="${BUILD_DIR}/tmp/deploy/licenses"
 
 IMAGE_BASENAME="core-image-pelux"
 SDK_BASENAME="pelux-glibc"
 
+#Enable globstar otion to recurses all the directories
+shopt -s globstar
+
 # If there is an QEMU directory
-if [ -d ${IMAGE_DIR}/${QEMU_ARCH} ]; then
+if [ -d ${IMAGE_DIR}/qemux86-64 ]; then
```

Comment:
why is this hardcoded now?

Reply:
It was hardcoded before. It's the only option left, so I decided to not use it

Reply:
use variable, please.